vcl_templates/www.vcl.erb

```diff
@@ -536,6 +564,32 @@ sub vcl_deliver {
 <% end # if -%>
   # End dynamic section
 
+if (req.http.Cookie ~ "cookies_policy") {
```

Comment:
I don't know how sensible it is to "merge" this `if` statement with the previous one starting on line 551

Reply:
I thought about it, it felt a bit eggy because it means mixing up the logic for this with something else - I figured that this keeps the code a bit more separated which (I think) makes the commit history/PR a bit more legible. Happy to change it if you want me to though

Reply:
I prefer to keep a single tree of logic. Anyway, let's wait for Bill to give his comments.